components/Collections/src/MDCCollectionViewFlowLayout.m

```diff
@@ -391,13 +391,28 @@ - (MDCCollectionViewLayoutAttributes *)updateSupplementaryViewAttribute:
   // view rows.
   CGRect insetFrame = attr.frame;
   if (!CGRectIsEmpty(insetFrame)) {
-    UIEdgeInsets insets = [self insetsAtSectionIndex:attr.indexPath.section];
+    UIEdgeInsets insets = self.sectionInset;
```

Comment:
Is the intention that the default insets be derived from -insetsAtSectionIndex? Currently this is not the case: the insets are only set to that value if the collectionView's delegate is nil (which is unlikely).

Perhaps the correct implementation would be to set the insets here to [self insetsAtSectionIndex] and conditionally override below if the flowLayoutDelegate responds to the selector which allows subclasses to customize the insets.